ophyd/signal.py

```diff
@@ -1283,7 +1283,14 @@ def limits(self):
         return (self._metadata["lower_ctrl_limit"], self._metadata["upper_ctrl_limit"])
 
     def _get_with_timeout(
-        self, pv, timeout, connection_timeout, as_string, form, use_monitor
+        self,
+        pv,
+        timeout,
+        connection_timeout,
+        count,
```

Comment:
I'm a bit torn, on one hand adding new positionals in the middle is a source of latent bugs, but on the other hand all of the positionals are required so any usage is fully broken (`TypeError: missing parameter`) rather than the subtle "all your inputs are shifted".

On net I think it is 👍🏻 , just set off alarms in my head.